### ISSUE TITLE
🚧 Daily Development Log: Optimizations & New Features

### ISSUE THREAD

**Issue by JinhuiYE** [starVLA/starVLA#64]
This issue serves as a living changelog to track our **daily development updates**.

We use this thread to document **performance optimizations** and **new features**. These changes are designed to enhance the project without altering the original core functionality (non-breaking changes).

### 🔔 Stay Updated
If you want to stay informed about the latest improvements as soon as they land, please **Subscribe** to this issue.

---

### 📋 Recent Updates
* **2026-04-26**: [Qwen3.5 Performance](https://github.com/starVLA/starVLA/issues/291)
* **2026-04-10 AMD GPU (ROCm) Support Verified**: Community-verified that starVLA runs out-of-box on AMD Instinct MI300X GPUs with ROCm 6.4, achieving results matching NVIDIA baselines with zero code changes. Added reference in LIBERO README. Details in [#254](https://github.com/starVLA/starVLA/issues/254).

* **2026-02-19 Training Speed References**: We've observed that video I/O can impact training speed differently across platforms. To help the community, we're now providing platform-specific speed benchmarks for reference #158.
* **2026-01-27 training cure for Florence**: Share training detail for FLorence VLM, https://github.com/starVLA/starVLA/issues/115
* **2026-01-20** **New VLM backbone**: Add support for nvidia cosmos-reason2, PR by @HaronW https://github.com/starVLA/starVLA/pull/111
* **2025-12-30** **Resume Training**: Added support for automatic resumption of training by detecting the latest checkpoint. [[commit]](https://github.com/starVLA/starVLA/commit/e1e6457c6cd124248f5ce7b2d3d40fb74f48c6fc)

*  **[2025/12/25]** **RoboCasa-G1 Bench**: Provided example for RoboCasa-G1-Tabletop. Reach SOTA without luxurious pre-training. Details in [examples/Robocasa_tabletop](https://github.com/starVLA/starVLA/tree/starVLA/examples/Robocasa_tabletop).
* **[2025/12/16]** **Loss curves shared**: Provided W&B links for pretrained checkpoints with complete loss vs. step plots. Details in [Issue #68](https://github.com/starVLA/starVLA/issues/68).
* **[2025/12/15]** **Auto-evaluation scripts released**: Added comprehensive [auto_eval_scripts](https://github.com/starVLA/starVLA/tree/starVLA/examples/SimplerEnv/eval_files/auto_eval_scripts) for obtaining full evaluation results automatically.
* **[2025/12/08]** Support Zero3 for larger batch sizes
* **[2025/12/02]** Support LeRobot dataset v3.0

**Comment by JinhuiYE** [comment]
# Wechat

<img width="90" height="146" alt="Image" src="https://github.com/user-attachments/assets/48a0fd4e-0e3b-4707-bdea-3283a265f60f" />


If outdated， find from RedBook.

<img width="98" height="134" alt="Image" src="https://github.com/user-attachments/assets/f262b2d2-86ee-4282-a600-5aed8bd79a37" />

**Comment by samuelwang23** [comment]
你好，微信群二维码好像已经失效了。方便的话能再发一个最新的二维码给我吗？谢谢啦！

**Comment by JinhuiYE** [comment]
> 你好，微信群二维码好像已经失效了。方便的话能再发一个最新的二维码给我吗？谢谢啦！

done

**Comment by XIEdeda** [comment]
这个群好像加不了了，满了200人、得群主邀请

**Comment by cheng-haha** [comment]
请问有第二个群吗

**Comment by bambooCV** [comment]
求拉

**Comment by zhenwang2002** [comment]
求拉

**Comment by 2870981470** [comment]
你好，微信群二维码好像已经失效了。方便的话能再发一个最新的二维码给我吗？谢谢啦！

**Comment by JinhuiYE** [comment]
> 你好，微信群二维码好像已经失效了。方便的话能再发一个最新的二维码给我吗？谢谢啦！

之前的 满了，新建了个

**Comment by Ethan-niu** [comment]
> Wechat

求拉微信群，谢谢

**Comment by hdzwxd-prog** [comment]
求拉微信群，感谢

**Comment by CapronerRevive** [comment]
求拉微信群，谢谢

**Comment by da2853** [comment]
Hi, the wechat QR code has expired. Could you please post a new code?

**Comment by hangqifan** [comment]
你好，微信群4月15日过期了。方便的话能再发一个最新的二维码给我吗？谢谢啦！

**Comment by Chenlingjing11** [comment]
你好，微信群过期了。能再发一个最新的二维码给我吗？感谢感谢

**Comment by wwwzxxzw** [comment]
求最新二维码

**Comment by monnv1** [comment]
请问有最新的二维码吗，感谢

**Comment by JinhuiYE** [comment]
done

**Comment by JoyDaDa** [comment]
求拉群

**Comment by yrnWang** [comment]
请问有最新的二维码吗，求拉群，感谢

**Comment by wangpaidan** [comment]
请问有最新的二维码吗？求拉，感谢！

**Comment by drilistbox** [comment]
求拉

<img width="130" height="216" alt="Image" src="https://github.com/user-attachments/assets/fc31df91-1321-4490-8137-299d8f279695" />

**Comment by sssssteve** [comment]
请问有新的二维码么，这个过期了

**Comment by JinhuiYE** [comment]
hi, If outdated， find from RedBook.

**Comment by JinhuiYE** [comment]
> # Wechat
> <img alt="Image" width="90" height="146" src="https://private-user-images.githubusercontent.com/131999494/615517714-48a0fd4e-0e3b-4707-bdea-3283a265f60f.JPG?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3ODQ3MTE1OTQsIm5iZiI6MTc4NDcxMTI5NCwicGF0aCI6Ii8xMzE5OTk0OTQvNjE1NTE3NzE0LTQ4YTBmZDRlLTBlM2ItNDcwNy1iZGVhLTMyODNhMjY1ZjYwZi5KUEc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjYwNzIyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI2MDcyMlQwOTA4MTRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04Mjk4M2VjNzU4NDdlY2Q3NTM0MjFjMjE0NjVkZmQxNDdiODJkZjkxMDJlMjMwOGMzYTgzYjI4Yzc0ZDg0Y2JhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZyZXNwb25zZS1jb250ZW50LXR5cGU9aW1hZ2UlMkZqcGVnIn0.qHXPkkMutrtthxNn0_HGXanrWP_R7ji5pLko6KqSmsw">
> If outdated， find from RedBook.
> 
> <img alt="Image" width="98" height="134" src="https://private-user-images.githubusercontent.com/131999494/581291646-f262b2d2-86ee-4282-a600-5aed8bd79a37.JPG?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3ODQ3MTE1OTQsIm5iZiI6MTc4NDcxMTI5NCwicGF0aCI6Ii8xMzE5OTk0OTQvNTgxMjkxNjQ2LWYyNjJiMmQyLTg2ZWUtNDI4Mi1hNjAwLTVhZWQ4YmQ3OWEzNy5KUEc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjYwNzIyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI2MDcyMlQwOTA4MTRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05ZDg3OGIzNTI4ODUyMzE1OTdkNTc5ZGYzMTcyNDZlOTVhOGJkNGU5MTdhYmFiZjMxZjUyYTllMGRlYzNkZDIzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZyZXNwb25zZS1jb250ZW50LXR5cGU9aW1hZ2UlMkZqcGVnIn0.dVXoCGsu6Likux84TSraJ7BOOkSjn9n24EfX27427HY">

hi， find from RedBook.
